src/app/event/_components/EventTitle.tsx

```diff
@@ -0,0 +1,26 @@
+import { CloverIcon } from '@/public/index';
+import classNames from 'classnames/bind';
+import { ReactNode } from 'react';
+import styles from './EventTitle.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface EventTitleProps {
+  title?: string;
+  children: ReactNode;
+  color: 'white' | 'black';
+}
+
+export default function EventTitle({ title, children, color }: EventTitleProps) {
+  return (
+    <div className={cn('title-wrap', { white: color === 'white', black: color === 'black' })}>
+      <div className={cn('title-top')}>
+        <CloverIcon fill={color === 'white' ? '#fff' : '#000'} />
```

Comment:
css로 처리하는 것은 어떨까요? 내부 white, black에 fill 넣으면 될 것 같아요...

```suggestion
        <CloverIcon className={cn(color)} />
```